Dockerfile

```diff
@@ -1 +1,7 @@
-FROM google/nodejs-runtime
+FROM node:14-alpine
```

Comment:
+1

Reply:
according to the CI failure it seems that git might not be available in the `alpine` variant.

Reply:
Yeah :(  Both slim and alpine caused problems here, so back to the full ubuntu image :/